samples/drivers/soc_flash_nrf/sample.yaml

```diff
@@ -4,3 +4,94 @@ tests:
   test:
```

Comment:
This need to be moved to become a test. The matching below does not work with sanitycheck and is error-prone.